packages/lodestar/test/perf/chain/validation/attestation.test.ts

```diff
@@ -26,3 +27,46 @@ describe("validate gossip attestation", () => {
     });
   }
 });
+
+describe.only("test throw AttestationError vs object", () => {
+  itBench({
+    id: "AttestationError",
+    fn: async () => {
```

Comment:
If you want to loop for 1000 times, use `runsFactor: 1000`. You should compare the results of looping 1000 times vs not looping. It's important to know the actual time of each operation.